pages/documentation.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/main.css">
+    <title>Components | FAST UI</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="left-nav">
+            <a href="#" class="nav nav-text">FAST UI</a>
+            <span class="small-text">V 1.0</span>
```

Comment:
<a href="#" />  isn;t it incorrect for the documentation page

Reply:
Done!!  fixed the path to index.html